Source/Flow/Public/Nodes/World/FlowNode_CallOwnerFunction.h

```diff
@@ -24,60 +26,64 @@ class FLOW_API UFlowNode_CallOwnerFunction : public UFlowNode
-	UPROPERTY(EditAnywhere, Category = "Call Owner", meta = (DisplayName = "Function"))
-	FFlowOwnerFunctionRef FunctionRef;
-
-	// Parameter object to pass to the function when called
-	UPROPERTY(EditAnywhere, Category = "Call Owner", Instanced)
-	UFlowOwnerFunctionParams* Params;
-
-protected:
-	// UFlowNode
-	virtual void ExecuteInput(const FName& PinName) override;
-	// ---
-
-	bool TryExecuteOutputPin(const FName& OutputName);
-	bool ShouldFinishForOutputName(const FName& OutputName) const;
+public:
 
 #if WITH_EDITOR
+	// UObject
+	virtual void PostLoad() override;
+	virtual bool CanEditChange(const FProperty* InProperty) const override;
+	virtual void PostEditChangeProperty(struct FPropertyChangedEvent& PropertyChangedEvent) override;
+	// --
 
-public:
 	// UFlowNode
 	virtual FText GetNodeTitle() const override;
-	virtual FString GetNodeDescription() const override;
-	virtual FString GetStatusString() const override;
 	virtual EDataValidationResult ValidateNode() override;
 
-	virtual bool SupportsContextPins() const override { return true; };
-	virtual TArray<FFlowPin> GetContextInputs() override;
-	virtual TArray<FFlowPin> GetContextOutputs() override;
-	// ---
-
-	// UObject
-	virtual void PostLoad() override;
-	virtual bool CanEditChange(const FProperty* InProperty) const override;
-	virtual void PostEditChangeProperty(struct FPropertyChangedEvent& PropertyChangedEvent) override;
-	// ---
+	virtual FString GetStatusString() const override;
+	// --
+#endif // WITH_EDITOR
 
-protected:
-	bool TryAllocateParamsInstance();
+	UFlowNode_CallOwnerFunction();
 
 	UClass* GetRequiredParamsClass() const;
 	UClass* GetExistingParamsClass() const;
 
-	static UClass* GetParamsClassForFunctionName(const UClass& ExpectedOwnerClass, const FName& FunctionName);
-	static UClass* GetParamsClassForFunction(const UFunction& Function);
-
-public:
 	bool IsAcceptableParamsPropertyClass(const UClass* ParamsClass) const;
 
 	UClass* TryGetExpectedOwnerClass() const;
+
 	static bool DoesFunctionHaveValidFlowOwnerFunctionSignature(const UFunction& Function);
 
+	static UClass* GetParamsClassForFunctionName(const UClass& ExpectedOwnerClass, const FName& FunctionName);
+	static UClass* GetParamsClassForFunction(const UFunction& Function);
+
 protected:
+
+#if WITH_EDITOR
+	void OnChangedParamsObject();
+#endif // WITH_EDITOR
+
+	// UFlowNode
+	virtual void ExecuteInput(const FName& PinName) override;
+	// --
+
+	bool ShouldFinishForOutputName(const FName& OutputName) const;
+	bool TryExecuteOutputPin(const FName& OutputName);
+
+	bool TryAllocateParamsInstance();
+
 	// Helper function for DoesFunctionHaveValidFlowOwnerFunctionSignature()
 	static bool DoesFunctionHaveNameReturnType(const UFunction& Function);
-#endif // WITH_EDITOR
+
+protected:
+
+	// Function reference on the expected owner to call
+	UE_DEPRECATED(5.4, "Sunsetting this feature from FlowGraph with UE 5.4.  Custom FlowNodes are a better mechanism to use.");
```

Comment:
As discussed, marking the CallOwnerFunction feature as Deprecated and scheduled for demolition.